ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -484,12 +484,11 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8
-- as: aws-ipi-disconnected-sts-basecap-none-addcap-f28
+- as: aws-ipi-disconnected-sts-basecap-none-f28
   cron: 21 14 14 * *
   steps:
     cluster_profile: aws-qe
     env:
-      ADDITIONAL_ENABLED_CAPABILITIES: CSISnapshot MachineAPI
```

Comment:
Are we going to update this job in this PR? If yes, I think we need to keep `MachineAPI` here.

Reply:
I thought installer only need the workaround for 4.14 installation? we also need it for 4.15 installation, right?

Reply:
ok, i'm not sure the difference between 4.14 and 4.15 behavior on the workaround, to save time, i will not touch install part in this pr, @jiajliu pls help handle it in dedicated pr for install part.

Reply:
> we also need it for 4.15 installation, right?
I think so.

Reply:
> pls help handle it in dedicated pr for install part.

okay.